src/componentLibrary/Modal/Modal.tsx

```diff
@@ -11,12 +11,7 @@ interface ModalProps {
   initialFocusRef?: MutableRefObject<HTMLElement | null>;
 }
 
-const Modal = ({
-  open,
-  onClose,
-  children,
-  initialFocusRef
-}: ModalProps): JSX.Element | null => {
+const Modal = ({ open, onClose, children, initialFocusRef }: ModalProps): JSX.Element | null => {
```

Comment:
@peterslany is linting/prettier running ok in your environment? I would have expected this to have been fixed on save when you pushed up the component (or possibly I've lost track of what was done when, and this preceded the prettier config changes :))

Reply:
@tomjeatt cc @crypto-engineer I think the on save formatting was reversed because it was in the PR with absolute imports. The config for the auto-formatting can be found here: https://github.com/interlay/interbtc-ui/blob/44fa8c78548fd409b66c5a3548a543ddb67dcb45/.vscode/settings.json
We may discuss whether we want to include it in main repo or just keep it locally as I think @nud3l was opposed to having this kind of config in the repository.